pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -539,6 +541,106 @@ func TestWithBinding(t *testing.T) {
 	}
 }
 
+func TestIsSchedulableAfterStorageClassAdded(t *testing.T) {
```

Comment:
please clean up the test cases too, we should be able to eliminate some test cases because we simplified the implementation.